packages/data-connect/src/api/index.ts

```diff
@@ -22,3 +22,11 @@ export * from './Mutation';
 export * from './query';
 export { setLogLevel } from '../logger';
 export { validateArgs } from '../util/validateArgs';
+
+export { DataConnectError, DataConnectOperationError } from '../core/error';
```

Comment:
Nit: these imports can be combined, right?

Reply:
Done